installation/routines/setup_git.sh

```diff
@@ -1,15 +1,88 @@
-#!/usr/bin/env bash
+convert_tardir_git_repo() {
+  echo "****************************************************"
+  echo "*** Converting tar-ball download into git repository"
+  echo "****************************************************"
+
+  git init -b "${GIT_BRANCH}"
+  git config pull.rebase false
+
+  # We always add origin as the selected (possible) user repository
+  # and MiczFlor's repository as upstream
+  # This means for developers everything is fully set up.
+  # For users there is no difference (origin = upstream = MiczFlor)
+  # We need to get the branch with larger depth, as we do not know
+  # how many commits happened between download and git repo init
+  if [[ $GIT_USE_SSH = true ]]; then
+      git remote add origin "git@github.com:${GIT_USER}/${GIT_REPO_NAME}.git"
+      git remote add upstream "git@github.com:MiczFlor/${GIT_REPO_NAME}.git"
```

Comment:
I would prefer to outsource `MiczFlor` to its own constant as well.
Maybe we could create a `00_constant.sh` where we store such matter, in addition to [this](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/installation/includes/01_default_config.sh#L3) 🤓